src/plugins/newsfeed/public/lib/api.ts

```diff
@@ -40,13 +41,23 @@ export interface NewsfeedApi {
 export function getApi(
   config: NewsfeedPluginBrowserConfig,
   kibanaVersion: string,
-  newsfeedId: string
+  newsfeedId: string,
+  isScreenshotMode: boolean
 ): NewsfeedApi {
-  const userLanguage = i18n.getLocale();
-  const fetchInterval = config.fetchInterval.asMilliseconds();
-  const mainInterval = config.mainInterval.asMilliseconds();
   const storage = new NewsfeedStorage(newsfeedId);
-  const driver = new NewsfeedApiDriver(kibanaVersion, userLanguage, fetchInterval, storage);
+  const mainInterval = config.mainInterval.asMilliseconds();
+
+  const createNewsfeedApiDriver = () => {
+    if (isScreenshotMode) {
+      return new NeverFetchNewsfeedApiDriver();
+    }
```

Comment:
Can we add a test in `src/plugins/newsfeed/public/lib/api.test.ts` to ensure the correct implementation is used depending on the `isScreenshotMode` input param?